### ISSUE TITLE
Add "wrapper" to NESTED_FUNCTIONS_WHITELIST?

### ISSUE THREAD

**Issue by pawamoy** [wemake-services/wemake-python-styleguide#1582]
# Rule request

## Thesis and reasoning

Decorators are often created inside functions. These decorators are then supposed to be used on top of other functions, which means they must accept one, and also that they will often create and return a new function by wrapping the original one. It's a level-3 nesting of functions. Maybe WPS430 should be relaxed, or `NESTED_FUNCTIONS_WHITELIST` could include another name like `wrapper`?

Code example:

```python
from functools import wraps


def python(versions):
    """Run through multiple Python versions."""

    def decorator(func):
        @wraps(func)
        def wrapper(context, *args, **kwargs):
            for version in versions:
                # do things with version and context
                func(context, *args, **kwargs)

        return wrapper

    return decorator


@python(["3.6", "3.7", "3.8"])
def task(context):
    ...
```

What do you think 🙂 ?

**Comment by sobolevn** [comment]
This might be a good one! 👍
